icicle-core/src/main/java/com/intenthq/icicle/IcicleIdGenerator.java

```diff
@@ -32,9 +30,9 @@
  *
  * We are generating an ID that will be comprised of the following:
  *
- * > 41 bit time + 10 bit logical shard id + 12 bit sequence id
+ * 41 bit time + 10 bit logical shard id + 12 bit sequence id
```

Comment:
Unfortunately, JavaDoc doesn't like the `>` at the start of this, so I removed it so it will actually publish to Maven Central without editing every time...